content/ADR-194-quests-poc-user-tracking.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: adr
+adr: 194
+title: Quests PoC - User tracking
+date: 2023-03-10
+status: Draft 
+type: RFC 
+spdx-license: CC0-1.0
+authors:
+  - guidota 
+---
+## Abstract
+
+This document proposes a protobuffer definition and implementation plan for a client-side quests system that can be used in the context of virtual worlds and games. The focus of this document is on user tracking and event processing.
+
+## Context, Reach & Prioritization
+
+The implementation of a quests system in a virtual world or game can greatly enhance the player experience and provide a sense of achievement and progress. This feature is particularly important for virtual worlds and games that have an open-world structure and rely on player-generated content. A client-side quests system can help guide players towards engaging with the available content and provide incentives for players to create their own content.
+
+## Solution Space Exploration
+
+The client will use the Decentraland RPC and protocol library to communicate with the QuestsService backend. (WebSockets + ProtoBuffers) 
+
+```proto
+syntax = "3";
+package decentraland.quests;
+
+message UserAddress {
+  string user_address = 1;
+}
+
+message StartQuestRequest {
+  string user_address = 1;
+  string quest_id = 2;
+}
+
+message StartQuestResponse {
+  /* There are a few valid reasons to not be accepted:
+  *  - Quest is not found
+  *  - Quest is deactivated (the owner deleted it)
+  *  - User already started the quest
+  *  - Internal errors (DB connection failed or something like that) */
+  bool accepted = 1;
+}
+
+message AbortQuestRequest {
+  string user_address = 1;
```

Comment:
is the user_address part of the payload? Shouldn't the user be already authenticated?